docker-compose.secrets.yml

```diff
@@ -43,6 +45,14 @@ services:
         secrets:
             # mounted at /run/secrets/scholarly-articles-aws-credentials
             - scholarly-articles-aws-credentials
+    web:
+        ports:
+            - ${PUBLIC_PORT_HTTPS:-8443}:443
```

Comment:
Does this mean that the `unstable` environment will only be available over HTTPS?

Reply:
This is meant to be merged with the other ports, so at the moment is should allow both 80 and 443. We can redirect-http-to-https later on.

Reply:
(once the HTTPS setup is proven to work)

Reply:
(I lose track of what merges and what replaces in Docker Compose.)